dependencies.gradle

```diff
@@ -37,9 +37,6 @@ def VERSIONS = [
         'info.ganglia.gmetric4j:gmetric4j:latest.release',
         'io.prometheus:simpleclient_common:latest.release',
         'io.prometheus:simpleclient_pushgateway:latest.release',
-        // TODO support HttpTagsProvider variants based on Jakarta APIs?
-//        'jakarta.servlet:jakarta.servlet-api:latest.release',
```

Comment:
I never made an issue for this, and no one has requested it (that I can remember now), but maybe we can open something to see if there's any interest. An issue is a better place than a comment here, anyway.